Sources/HaCWebsiteLib/Views/LandingPage/LandingFeatures/LandingFeatures.swift

```diff
@@ -23,11 +23,8 @@ enum LandingFeatures {
 
     // We want to return the first non-nil LandingFeature whos expiry date is
```

Comment:
Is this comment necessary?

Reply:
I think so, because we want our projects to be accessible as possible. Even if this comment would be a bit redundant to those familiar with Swift, a lot of programmers on this project may not be, so for more complex things a comment could be very helpful.

I would agree, but it's something that I wouldn't have understood unless I knew the problem that needed solving.